control-plane/stor-port/src/types/v0/store/pool.rs

```diff
@@ -134,6 +190,12 @@ impl SpecTransaction<PoolOperation> for PoolSpec {
                 PoolOperation::Create => {
                     self.status = SpecStatus::Created(transport::PoolStatus::Online);
                 }
+                PoolOperation::Label(PoolLabelOp { labels, .. }) => {
```

Comment:
there's another trait or method in this trait where we indicate if the operation needs to be a transaction - we should update that too since here we don't need it

Reply:
Added this 
```
  fn log_op(&self, _operation: &PoolOperation) -> (bool, bool) {
        (false, false)
    }
    ```